bolt-modules/boltlib/lib/puppet/functions/puppetdb_command.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require 'bolt/error'
+
+# Send a command with a payload to PuppetDB.
+#
+# The `pdb_command` function only supports version 5 of the `replace_facts`
+# command. Other commands might also work, but are not tested or supported
+# by Bolt.
+#
+# See the [commands endpoint](https://puppet.com/docs/puppetdb/latest/api/command/v1/commands.html)
+# documentation for more information about available commands and payload
+# format.
+#
+# _This function is experimental and subject to change._
+#
+Puppet::Functions.create_function(:puppetdb_command) do
+  # @param command The command to invoke.
+  # @param version The version of the command to invoke.
```

Comment:
```suggestion
  # @param version The version of the API endpoint to target.
```

Reply:
This is taken directly from the PDB documentation: https://puppet.com/docs/puppetdb/latest/api/command/v1/commands.html

```
version is a JSON integer describing what version of the given command you’re attempting to invoke.
```